src/parquet/file/writer.cc

```diff
@@ -55,8 +55,8 @@ ParquetFileWriter::~ParquetFileWriter() {
 
 std::unique_ptr<ParquetFileWriter> ParquetFileWriter::Open(
     std::shared_ptr<OutputStream> sink, std::shared_ptr<GroupNode>& schema,
-    MemoryAllocator* allocator, const std::shared_ptr<WriterProperties>& properties) {
-  auto contents = FileSerializer::Open(sink, schema, allocator);
+    MemoryAllocator* allocator, const std::shared_ptr<WriterProperties>& props) {
```

Comment:
Saving only a tiny number of characrters is not really a benefit, we rather stick to the full word for better code readability.